src/Addons/34thPRC_ArmourStandard/data/marines/config_weapons.hpp

```diff
@@ -17,6 +31,19 @@ class 34thPRC_ArmourStandard_Marines_CH252_ArcticLight : 34thPRC_ArmourStandard_
 		"34thPRC_ArmourStandard\data\marines\arctic\helmet_arctic_light_co.paa"
 	};
 }
+class 34thPRC_ArmourStandard_Marines_CH252_ArcticLight_NV : 34thPRC_ArmourStandard_Marines_CH252_Urban_NV
+{
+	displayName="[34th] CH252 Arctic Light (NV)";
+	author="TheBrwnKidd & Vazya";
+	hiddenSelections[]=
+	{
+		"_Visor"
+	};

```

Comment:
I believe that we don't need this part to be included again as we can inherit it from `34thPRC_ArmourStandard_Marines_CH252_Urban_NV`. We possibly could even be inheriting it from `HaloInf_Marine_DES_UNSC_NV_headgear`.

Reply:
Cleaned up the classes to inherit "_Visor" from HaloInf_Marine_DES_UNSC_NV_headgear.